yamllint/cli.py

```diff
@@ -31,7 +31,7 @@ def find_files_recursively(items, conf):
             for root, dirnames, filenames in os.walk(item):
                 for f in filenames:
                     filepath = os.path.join(root, f)
-                    if conf.is_yaml_file(filepath):
+                    if conf.is_yaml_file(filepath) and not conf.is_file_ignored(filepath):
```

Comment:
This check is also performed in `linter.run`. See https://github.com/adrienverge/yamllint/pull/524#discussion_r1052261762

Reply:
About https://github.com/adrienverge/yamllint/pull/524#discussion_r1052261762, @andrewimeson is right: it would have broken the Python lib usage.

However it looks risky to modify `find_files_recursively()`. What about testing `conf.is_file_ignored(filepath)` in the for loop of `run()` instead, to mimic the standard usage as closely as possible?

Reply:
@adrienverge Done :heavy_check_mark: 

Note that initially I was confused by "loop of `run()` because it wasn't clear to me which `run` is meant. I opted for the `run` function in `cli.py` :sweat_smile:

Reply:
Oops, sorry I wasn't clear! But you guessed right: I meant the `run()` of `cli.py` :+1: